public/mall/account/register/index.html

```diff
@@ -68,7 +69,164 @@
 </header>
 
 <main>
-  To be implemented
```

Comment:
This will cause conflict with your other branches in future pull request. Please remove everything inside of the `main` tag